hyp3_gamma/__main__.py

```diff
@@ -183,6 +184,13 @@ def water_map():
     parser.add_argument('--hand-fraction', type=float, default=0.8)
     parser.add_argument('--membership-threshold', type=float, default=0.45)
     parser.add_argument('granule')
+
+    parser.add_argument('--include-flood-depth', type=string_is_true, default=False)
+    parser.add_argument('--estimator', type=str, default='nmad', choices=['iterative', 'logstat', 'nmad', 'numpy'])
+    parser.add_argument('--water-level-sigma', type=float, default=3., )
+    parser.add_argument('--known-water-threshold', type=float, default=30.)
+    parser.add_argument('--iterative-bounds', type=int, nargs=2, default=[0, 15])
```

Comment:
were the bounds intended to be `int` or `float`? we called them floats in the hyp3 PR

Reply:
asf-tools has it as an `int`, so this looks good, and we should update the type in `hyp3`
https://github.com/ASFHyP3/asf-tools/blob/develop/asf_tools/flood_map.py#L126

Reply:
yup, int. Would that just be updating '0.0' and '15.0' to '0' and '15'?